src/components/directory-content.js

```diff
@@ -342,28 +342,28 @@ const DirectoryContent = () => {
         );
     }
 
-    function creatorCellRender(cellData) {
-        const creator = cellData.rowData[cellData.dataKey];
+    function userCellRender(cellData) {
+        const user = cellData.rowData[cellData.dataKey];
         return (
             <div className={classes.cell}>
-                <Tooltip title={creator} placement="right">
+                <Tooltip title={user} placement="right">
                     <Chip
                         className={classes.chip}
-                        label={abbreviationFromUserName(creator)}
+                        label={abbreviationFromUserName(user)}
                     />
                 </Tooltip>
             </div>
         );
     }
 
-    function createdCellRender(cellData) {
-        const created = new Date(cellData.rowData[cellData.dataKey]);
-        if (created instanceof Date && !isNaN(created)) {
-            const date = created.toLocaleDateString(intl.locale);
+    function dateCellRender(cellData) {
+        const data = new Date(cellData.rowData[cellData.dataKey]);
```

Comment:
Calling it data instead of date is deliberate ? What is your train of thought, since you renamed the renderer to `dateCellRender`

Reply:
there is a date variable two lines below, so I thought it was ok here